Bruin-SuperMileage/EV
# PR Template

# Description
The following file is a PR template for this repo. This gives us a standard way to document proposed changes to the repo and make reviewing PRs easier. WIth this change, every time you open a PR it will fill the 'description' part of the PR documentation with the markdown text written in `.github/pull_request_template.md` which is what you are viewing now.

For more information on pull request templates or how it was implemented, view the Github docs [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

## Type of change

# How Has This Been Tested?
The only tests done on this change was to:
1. Fork the EV repo
2. Create `.github/pull_request_template.md` in an new branch
3. Merge the branch to main on the fork
4. Make another generic PR and make sure that it fills the description with the template

See verification of that on my fork [here](https://github.com/IChocked/EV) and feel free to make test pull requests to verify this on the fork

# Checklist:
